pkg/garden/sur/treaty.hoon

```diff
@@ -18,6 +18,20 @@
         [%del =ship =desk]
     ==
   --
+:: TODO gotta be a better way to do this
+::
+++  treaty-0
+  =<  treaty
+  |%
+  +$  treaty
+    [=ship =desk =case hash=@uv =docket-0:docket]
+  ::
+  +$  update
+    $%  [%ini init=(map [=ship =desk] =treaty)]
+        [%add =treaty]
```

Comment:
Hmm good q. I guess you can't `treaty(docket:docket docket-0)`... Pretty sure I've seen something weird be done for cases like this, but can't remember what it was.